sdk/core/azure-core/dev_requirements.txt

```diff
@@ -1,5 +1,6 @@
 trio; python_version >= '3.5'
 aiohttp>=3.0; python_version >= '3.5'
+isodate>=0.6.0
```

Comment:
This is only a test requirement right? Core isn't taking a dependency on this library?

Reply:
Yep, this would only be for testing. I could also try to work `isodate` out of the tests -- it's convenient but not necessary